spec.bs

```diff
@@ -146,26 +146,26 @@ The RDF expansion is defined for each property in the `cdt:Map` by a `tss:Path`:
     { 
       "time": "2026-01-01T06:00:00Z",
       "value": "{
-        "latitude": "50.000000"^^xsd:double,
-        "longitude": "4.000000"^^xsd:double,
-        "temperature": "5.21"^^xsd:double
-      }"^^cdt:Map
+        "latitude": "50.000000"^^<http://www.w3.org/2001/XMLSchema#double>,
+        "longitude": "4.000000"^^<http://www.w3.org/2001/XMLSchema#double>,
+        "temperature": "5.21"^^<http://www.w3.org/2001/XMLSchema#double>
+      }"^^<http://w3id.org/awslabs/neptune/SPARQL-CDTs/Map>
     },
     { 
       "time": "2026-01-01T07:00:00Z",
       "value": "{
-        "latitude": "50.100000"^^xsd:double,
-        "longitude": "4.100000"^^xsd:double,
-        "temperature": "7.61"^^xsd:double
-      }"^^cdt:Map
+        "latitude": "50.100000"^^<http://www.w3.org/2001/XMLSchema#double>,
+        "longitude": "4.100000"^^<http://www.w3.org/2001/XMLSchema#double>,
+        "temperature": "7.61"^^<http://www.w3.org/2001/XMLSchema#double>
+      }"^^<http://w3id.org/awslabs/neptune/SPARQL-CDTs/Map>
     },
     { 
       "time": "2026-01-01T08:00:00Z",
       "value": "{
-        "latitude": "50.200000"^^xsd:double,
-        "longitude": "4.200000"^^xsd:double,
-        "temperature": "4.21"^^xsd:double
-      }"^^cdt:Map
+        "latitude": "50.200000"^^<http://www.w3.org/2001/XMLSchema#double>,
+        "longitude": "4.200000"^^<http://www.w3.org/2001/XMLSchema#double>,
+        "temperature": "4.21"^^<http://www.w3.org/2001/XMLSchema#double>
+      }"^^<http://w3id.org/awslabs/neptune/SPARQL-CDTs/Map>
     }
   ]"^^cdt:List;
```

Comment:
To be on the same page: this doesn't need to be expanded because it is not inside a `cdt` data type right?

Reply:
Indeed